src/catfs/rlibc.rs

```diff
@@ -460,6 +493,11 @@ impl File {
         }
     }
 
+    #[cfg(target_os = "macos")]
+    pub fn allocate(&self, offset: u64, len: usize) -> io::Result<()> {
+        self.truncate((offset as usize)+ len)
```

Comment:
This should ideally be replaced with some `fcntl` stuff. See https://stackoverflow.com/a/11497568/3770260.

But sure, we are all using SSD these days so speed isn't a concern. And Apple products don't really have longivity concerns.

Reply:
This PR is superseded by #45.  Could you review that instead?

Reply:
Sure.